lib/mutant/mutator/node/regexp/greedy_zero_or_more.rb

```diff
@@ -0,0 +1,24 @@
+module Mutant
+  class Mutator
+    class Node
+      module Regexp
+        # Mutator for greedy zero-or-more quantifier, `*`
+        class GreedyZeroOrMore < Node
+          handle(:regexp_greedy_zero_or_more)
+
+          children :min, :max, :subject
+
+          # Emit mutations
+          #
+          # Replace `/a*/` with `/a+/`
+          #
+          # @return [undefined]
+          def dispatch
+            emit(s(:regexp_greedy_one_or_more, *children))
+            emit_subject_mutations
```

Comment:
Apparently `min` and `max` are `Integer`s and `emit_min_mutations` doesn't work:

```
     Failure/Error: self::REGISTRY.lookup(node.type).call(node, parent)

     NoMethodError:
       undefined method `type' for 0:Fixnum
```

I'm happy to add mutations for these at some point but I assume I have to do something custom to get this to work.